sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java

```diff
@@ -39,6 +39,9 @@
     private String since;
     private String deprecated;
 
+    // The groups below are almost the same with ones defined in `sql/functions.sql`.
+    // But, `collection_funcs` is split into fine-grained three groups: `array_funcs`, `map_funcs`,
+    // and `json_funcs`.
```

Comment:
This is a minor follow-up for #28224, @HyukjinKwon

Reply:
Nice but what about we document this in `ExpressionDescription`, and here we say things like please refer `ExpressionDescription`?

Reply:
Yea, the idea looks nice. I'll update.